docs/guide/partition.md

```diff
@@ -76,22 +76,56 @@ If KD-tree column doesn't exist, it's automatically added.
 
 ## By Admin Boundaries
 
-Split by country code or other administrative column:
+Split by administrative boundaries via spatial join with remote datasets:
+
+### How It Works
+
+This command performs **two operations**:
+
+1. **Spatial Join**: Queries remote admin boundaries using spatial extent filtering, then spatially joins them with your data
+2. **Partition**: Splits the enriched data by administrative levels
+
+### Quick Start
 
 ```bash
-# Preview partitions
-gpio partition admin input.parquet --preview
+# Preview GAUL partitions by continent
+gpio partition admin input.parquet --dataset gaul --levels continent --preview
+
+# Partition by continent
+gpio partition admin input.parquet output/ --dataset gaul --levels continent
+
+# Hive-style partitioning
+gpio partition admin input.parquet output/ --dataset gaul --levels continent --hive
+```
+
+### Multi-Level Hierarchical Partitioning
+
+Partition by multiple administrative levels:
+
+```bash
+# Hierarchical: continent → country
+gpio partition admin input.parquet output/ --dataset gaul --levels continent,country
 
-# Partition by default column (admin:country_code)
-gpio partition admin input.parquet output/
+# All GAUL levels: continent → country → department
+gpio partition admin input.parquet output/ --dataset gaul --levels continent,country,department
 
-# Custom admin column
-gpio partition admin input.parquet output/ --column iso_code
+# Hive-style multi-level (creates continent=Africa/country=Kenya/department=Accra/)
+gpio partition admin input.parquet output/ --dataset gaul \
+    --levels continent,country,department --hive
 
-# Hive-style
-gpio partition admin input.parquet output/ --hive
+# Overture Maps by country and region
+gpio partition admin input.parquet output/ --dataset overture --levels country,region
```

Comment:
What happens if I want both gaul and overture? Like continent, country, department and region? My guess is that the second one run will overwrite the first one for the country? But the ones that area in the first and not the second will stay as the first? 

I think that can be fine, but we should document it - I think some people may be interested in getting multiple admin levels.